dot/state/epoch.go

```diff
@@ -296,6 +304,10 @@ func (s *EpochState) setLatestConfigData(epoch uint64) error {
 // If the header params is nil then it will search only in the database.
 func (s *EpochState) GetConfigData(epoch uint64, header *types.Header) (configData *types.ConfigData, err error) {
 	for tryEpoch := int(epoch); tryEpoch >= 0; tryEpoch-- {
+		if tryEpoch == 0 {
```

Comment:
actually not, before getting epoch data or config data we don't check the epoch, the check is inside both functions, and this check is under the loop bc the ConfigData is not present in every epoch, so if the runtime issues a config data in epoch 1 it will use this same config for many epochs, each time we want to get a config data we should use the latest we know for that chain